README.md

```diff
@@ -20,21 +20,21 @@ Use the available commands to manage your contacts and connect with students in
 ### Command summary
 Action | Format, Examples
 | --- | --- |
-Add	| `add n/NAME y/YEAR c/COURSE e/EMAIL [l/LINKEDIN] [g/GITHUB] [m/YEARTAKEN MODCODE]... [t/TAG]...`<br />e.g., `add n/Bob Sim y/2 c/com sci e/e0823741@nus.edu l/linkedin.com/in/bom-sim-086g93847/ m/ay2223s2 cs2103t m/ay2223s2 cs2101 t/python t/java`
+Add	| `add n/NAME y/YEAR c/COURSE e/EMAIL [l/LINKEDIN] [g/GITHUB] [m/YEARTAKEN MODCODE]... [s/SKILL]...`<br />e.g., `add n/Bob Sim y/2 c/com sci e/e0823741@nus.edu l/linkedin.com/in/bom-sim-086g93847/ m/ay2223s2 cs2103t m/ay2223s2 cs2101 s/python s/java`
 View contact | `view INDEX`<br />e.g., `view 3`
 View tab | `view C/M/T`<br />e.g., `view c`, `view m` or `view t`
-Edit | `edit [n/NAME] [y/YEAR] [c/COURSE] [e/EMAIL] [l/LINKEDIN] [g/GITHUB] [m/MODINDEX YEARTAKEN MODCODE]... [t/TAG]...`<br />e.g., `edit n/Bob Lim m/2 ay2223s2 cs2109s`
+Edit | `edit [n/NAME] [y/YEAR] [c/COURSE] [e/EMAIL] [l/LINKEDIN] [g/GITHUB] [m/MODINDEX YEARTAKEN MODCODE]... [s/SKILL]...`<br />e.g., `edit n/Bob Lim m/2 ay2223s2 cs2109s`
 Find	name | `find NAME [MORENAMES]`<br />e.g., `find Bob Sam Pete`
 Find module | `findm MODCODE [MOREMODCODES]`<br />e.g., `findm cs2013t cs2109s cs2107`
-Find tags | `findt TAG [MORETAGS]`<br />e.g., `findt java python` 
+Find skills | `findt SKILL [MORESKILLS]`<br />e.g., `findt java python` 
```

Comment:
Should `findt` be implemented as `finds`?

Reply:
Fixed, thanks for pointing that out 👍